<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication1Class</class>
 <widget class="QMainWindow" name="QtGuiApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>249</width>
       <height>199</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>249</width>
       <height>219</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>290</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>处理图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>打开的原图</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>打开的处理图</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenCameraBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseCameraBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>440</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TakePicBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>490</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_open_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>926</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_process_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>792</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>924</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenCameraBtn</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_OpenCameraBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>918</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CloseCameraBtn</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_CloseCameraBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>910</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TakePicBtn</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiApplication1Class</receiver>
   <slot>on_TakePicBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>781</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>910</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_TakePicBtn_clicked()</slot>
 </slots>
</ui>
